jwql/jwql_monitors/generate_preview_images.py

```diff
@@ -633,12 +633,18 @@ def process_program(program):
         The program identifier (e.g. ``88600``)
     """
 
-    # Group together common exposures
     logging.info('')
     logging.info('Processing {}'.format(program))
+
+    # Gather files to process
     filenames = glob.glob(os.path.join(get_config()['filesystem'], 'public', program, '*/*.fits'))
     filenames.extend(glob.glob(os.path.join(get_config()['filesystem'], 'proprietary', program, '*/*.fits')))
     filenames = list(set(filenames))
+
+    # Ignore "original" files
+    filenames = [item for item in filenames if '_original.fits' not in item]
```

Comment:
This is needed because the DMS build filesystem contains a plethora of `_original.fits` filenames, which I don't think will ultimately be present in the MAST archive, and are not needed by the JWQL app.

Reply:
yeah I agree, I don't think these are important